Sources/Core/Renderer/GUI/GUIElement/GUIList.swift

```diff
@@ -4,11 +4,13 @@ struct GUIList: GUIElement {
   var items: [GUIListItem]
   var rowHeight: Int
   var renderRowBackground: Bool
+  var rightAlign: Bool
 
-  init(rowHeight: Int, renderRowBackground: Bool = false) {
+  init(rowHeight: Int, renderRowBackground: Bool = false, rightAlign: Bool = false) {
```

Comment:
Instead of having a `rightAlign` boolean, it'd be better to have an `alignment` parameter with type `Alignment`, where `Alignment` is just an enum with two cases, left and right. The default can be .left instead of false. The `Alignment` enum can just be declared within the `GUIList` type just before `init`. This nests the type and means that to anyone outside of `GUIList` it's called `GUIList.Alignment` which makes it super clear where it's used. lmk if that didn't make sense